src/sentry/notifications/helpers.py

```diff
@@ -295,6 +296,13 @@ def get_scope(
 def get_target_id(user: User | None = None, team: Team | None = None) -> int:
     """:returns the actor ID from a User or Team."""
     if user:
+        # TODO(hybrid-cloud): Dual write the user actor_id for now. This will get removed later.
```

Comment:
Hmm, I'm not sure this will work.  `get_target_id` runs in control silo endpoints a lot right now (making all kinds of integrations decisions), so creating the `Actor` here may be risky.

Should creating the actors here be a service method instead?  I think that will be correct in our final implementation anyways.